ntfysh_client/MainForm.cs

```diff
@@ -409,5 +411,32 @@ private void exitToolStripMenuItem1_Click(object sender, EventArgs e)
             _trueExit = true;
             Close();
         }
+
+        private async void SendNotificationMenuItem_Click(object sender, System.EventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async void notificationTopics_MouseDoubleClick(object sender, MouseEventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async Task SendMessage()
+        {
+            if (notificationTopics.SelectedItem != null && string.IsNullOrEmpty(notificationTopics.SelectedItem as string) == false)
+            {
+                var dlg = new SendMessageForm();
+                var r = dlg.ShowDialog(this);
+                if (r == DialogResult.OK)
+                {
+                    var topicAndHost = notificationTopics.SelectedItem.ToString()?.Split("@");
```

Comment:
It appears with the null conditional ? that `topicAndHost` could be null resulting in a `NullReferenceException` in the following two lines should there be an issue. Please could you verify whether `ToString()`can return null, and if so, handle the condition with an error dialog box accordingly?